src/macros/macrorecorder.cpp

```diff
@@ -0,0 +1,106 @@
+#include "macrorecorder.h"
+
+#include <QtConcurrentRun>
+
+#include "control/controlproxy.h"
+#include "preferences/configobject.h"
+
+namespace {
+constexpr uint kMaxMacroSize = 1000;
```

Comment:
The constant is also used literally for the size of the SPSC queue -> DRY

Reply:
Also, I don't think we use `uint` in our code base. I think we should rather use either `unsigned int` or `quint`.

Reply:
@uklotzde then I need to move it back to a class constant, since this is in the cpp file?

Reply:
Of course. I assume basic C++ knowledge and will not describe individual steps on how to achieve a proposed goal. Often alternative solutions are possible and valid. It's your task to figure out the details.

Reply:
My point is that you previously suggested to move it from the class constant into the namespace.
I don't need step by step instructions, but apart from a class constant or a constant in the mixxx namespace I can't think of alternative solutions right now.

Reply:
Are you talking about the SPSCQueue initialized in EngineBuffer? That one is for *playback*, right? So these are two different things and the consts should be independent. If @xerus2000 implements the periodical fetching of actions, the recording queue can be much smaller.